README.md

```diff
@@ -12,13 +12,13 @@ The contents of this code of conduct apply to our interactions in various areas
 
 ## Values
 
-We want our team to be a fun, productive, and safe space for all members. In addition to core values of respect and honesty, there are several ways in which our product team distinguishes itself.
+We want our team to be fun, productive, and safe for all members. In addition to the Social Tables [core values](https://www.socialtables.com/company/about/#values), there are several ways in which our Engineering team distinguishes itself.
```

Comment:
What I've presented would be Engineering's CoC. I think there's value in that, but I've been wondering whether it should be expanded to Product, as well (which would encompass Design).

It seems like you're suggesting that you think it should include Product. Is that correct? We would need to ask them if they want to participate.